src/models/assets/Stock.py

```diff
@@ -1,9 +1,9 @@
 # import Asset
 
-# This object has to match the API being used in order to make json.dumps() 
+# This object has to match the API being used in order to make json.dumps()
 # work
-class Stock(Asset):
-    def __init__(self, open: float, close: float, high: float, low: float, average_volume: float, market_cap: float, peratio: float, dividend_yield: float, asset_type: string, last: float, symbol: string, prev_close: float):
+class Stock():
```

Comment:
removed `Asset` as it doesn't seem to be used for now. Please add when neccessary

`string` -> `str`